docs/release-notes/1.1/release-1.1.md

```diff
@@ -0,0 +1,211 @@
+# .NET for Apache Spark 1.1 Release Notes
+
```

Comment:
Can you add a deprecation notice section? https://github.com/dotnet/spark/issues/839

Reply:
Done!